tardis/simulation/tests/test_simulation.py

```diff
@@ -172,3 +173,95 @@ def test_logging_simulation(atomic_data_fname, caplog):
 
     for record in caplog.records:
         assert record.levelno >= logging.INFO
+
+
+# Testing Configuration of Logger via run_tardis() Function
+# TODO: Move to its own Testing Module (Logger)
+@pytest.mark.parametrize(
+    ["log_state", "specific"],
+    [
+        ("Info", False),
+        ("inFO", False),
+        ("info", False),
+        ("InFo", False),
+        ("INFO", False),
+        ("INFO", True),
+        ("DEBUG", False),
+        ("DEBUG", True),
+        ("WARNING", False),
+        ("WARNING", True),
+        ("ERROR", False),
+        ("ERROR", True),
+        ("CRITICAL", False),
+        ("CRITICAL", True),
+        ("NOTSET", False),
+        ("NOTSET", True),
+    ],
+)
+def test_logging_config(atomic_data_fname, caplog, log_state, specific):
+    config = Configuration.from_yaml(
+        "tardis/io/tests/data/tardis_configv1_verysimple.yml"
+    )
+    config["atom_data"] = atomic_data_fname
+
+    log_states = {
+        "NOTSET": logging.NOTSET,
+        "DEBUG": logging.DEBUG,
+        "INFO": logging.INFO,
+        "WARNING": logging.WARNING,
+        "ERROR": logging.ERROR,
+        "CRITICAL": logging.CRITICAL,
+    }
+    caplog.clear()
+    run_tardis(config=config, log_state=log_state, specific=specific)
+    for record in caplog.records:
+        if specific == True:
+            assert record.levelno == log_states[log_state.upper()]
+        else:
+            assert record.levelno >= log_states[log_state.upper()]
+
+
+@pytest.mark.parametrize(
+    ["log_state", "specific"],
+    [
+        ("Info", False),
```

Comment:
this seems to be a repeat of the parameterize before (in the sense of it should be in a variable

Reply:
Yes, I was thinking about this. I think I can make it such that it accessible to both the cases. 
I will also add another test when both are specified ie Functional argument & YAML property 😄

Reply:
Yes, made it available for mutliple uses 😉